drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -140,8 +327,8 @@ static int get_device_type(const struct device *dev, uint16_t *val)
  *
  * @return -ENOTSUP for unsupported channels
  */
-static int channel_get(const struct device *dev, enum sensor_channel chan,
-		       struct sensor_value *val)
+static int bq274xx_channel_get(const struct device *dev, enum sensor_channel chan,
```

Comment:
Please remove the comment block above since it is already in `sensor.h`

Reply:
Oh yeah, I'm all for less comments. :-)